build.gradle

```diff
@@ -143,13 +143,19 @@ project('client') {
         compile group: 'org.projectlombok', name: 'lombok', version: lombokVersion
         compile group: 'com.google.code.findbugs', name: 'annotations', version: findbugsVersion
         compile group: 'io.netty', name: 'netty-all', version: nettyVersion
+        compile "io.grpc:grpc-netty:" + grpcVersion
+        compile "io.grpc:grpc-all:" + grpcVersion
+        compile "io.grpc:grpc-protobuf:" + grpcVersion
+        compile "io.grpc:grpc-stub:" + grpcVersion
         compile project(':common')
         compile project(':shared')
         compile project(':shared:protocol')
         compile project(":shared:controller-api")
         testCompile project(':test:testcommon')
         testCompile group: 'org.slf4j', name: 'log4j-over-slf4j', version: slf4jApiVersion
         testCompile group: 'ch.qos.logback', name: 'logback-classic', version: qosLogbackVersion
+        // https://mvnrepository.com/artifact/org.jasypt/jasypt
+        compile group: 'org.jasypt', name: 'jasypt', version: '1.9.2'
```

Comment:
Why are we pulling in a dependency to do SHA-1 based salted encryption when PBKDF2 using Hmac-SHA-1 is built into Java and considered cryptographically stronger?
https://howtodoinjava.com/security/how-to-generate-secure-password-hash-md5-sha-pbkdf2-bcrypt-examples/

Reply:
Thanks for the link. Will look at this and figure out if we really need the external dependency.